ingest/ingest_pipeline.py

```diff
@@ -94,6 +95,7 @@
     from .expression_files.dense_ingestor import DenseIngestor
     from .expression_files.mtx import MTXIngestor
     from .cli_parser import create_parser, validate_arguments
+    from .de import DifferentialExpression, prepare_h5ad
```

Comment:
This looks unused in this module, so let's remove it.  

Otherwise, I think we'd need to also include it in the import block above this one.

```suggestion
    from .de import DifferentialExpression
```

Reply:
Thanks for catching that! I had removed from the top import block and neglected the second. `5928217`